configs/config.json.example

```diff
@@ -22,12 +22,12 @@
     "distance_unit": "km",
     "reconnecting_timeout": 15,
     "item_filter": {
-      "1": { "keep" : 100 },
-      "101": { "keep" : 10 },
-      "102": { "keep" : 30 },
-      "103": { "keep" : 30 },
-      "201": { "keep" : 30 },
-      "701": { "keep" : 100 }
+      "Pokeball":       { "keep" : 100 },
```

Comment:
Added a check before it even connects to server (for faster feedback), hence the reason for the json.load